tests/integration/tests/test_ingress.py

```diff
@@ -41,7 +41,7 @@ def get_ingress_service_node_port(p):
 def get_external_service_ip(instance: harness.Instance, service_namespace) -> str:
     try_count = 0
     ingress_ip = None
-    while ingress_ip is None and try_count < 5:
+    while ingress_ip is None and try_count < 20:
```

Comment:
The changes here are not leftovers from testing right? Just making sure these are intentional.

Reply:
These changes are intentional, aiming to reduce test flakiness.